docker-compose.yml

```diff
@@ -64,3 +64,16 @@ services:
     depends_on:
       - mqtt
     restart: unless-stopped
+
+  edge-detect:
```

Comment:
Keep the original docker-compose.yml as-is until we've put the ML components through their paces.  Add this content into a new `docker-compose-edge.yml` for us to do system testing on for a couple weeks to ensure stability before we release this as the formal docker-compose.

Also, this only is primed for one of the two ML models.  Add in both

Reply:
Removed from repo. Built new docker compose file.